plugins/biolatency/src/index.tsx

```diff
@@ -0,0 +1,259 @@
+import { Plugin } from '@kinvolk/headlamp-plugin/types/plugin/index.d';
+import Registry from '@kinvolk/headlamp-plugin/types/plugin/registry.d';
+import playCircleOutline from '@iconify/icons-mdi/play-circle-outline';
+import stopIcon from '@iconify/icons-mdi/stop';
+import restartIcon from '@iconify/icons-mdi/restart';
+const pluginLib = window.pluginLib;
+const { Loader } = window.pluginLib.CommonComponents;
+const React = window.pluginLib.React;
+const { BarChart, XAxis, YAxis, Tooltip, Legend, Bar } = window.pluginLib.Recharts;
+const K8s = pluginLib.K8s.ResourceClasses;
+const { Box, IconButton, Typography, Button } = pluginLib.MuiCore;
+const { Icon } = pluginLib.Iconify;
+
+// const BIOLATENCY_CONTAINER_WIDTH = '500px';
+// const BIOLATENCY_CONTAINER_HEIGHT = '500px';
+const TRACES_RESOURCE_GROUP = 'gadget.kinvolk.io';
+const IG_TRACE_NAMESPACE = 'gadget';
+const TRACE_BIOLATENCY_RESOURCE_NAME = 'biolatency';
+const ResourceAPI = new K8s.Resource(TRACES_RESOURCE_GROUP, 'v1alpha1', 'traces');
+
+function startLatencyCheck() {
+  return ResourceAPI.patch(
+    [{ op: 'add', path: '/metadata/annotations/gadget.kinvolk.io~1operation', value: 'start' }],
+    IG_TRACE_NAMESPACE,
+    TRACE_BIOLATENCY_RESOURCE_NAME
+  );
+}
+
+function stopLatencyCheck() {
+  return ResourceAPI.patch(
+    [{ op: 'add', path: '/metadata/annotations/gadget.kinvolk.io~1operation', value: 'stop' }],
+    IG_TRACE_NAMESPACE,
+    TRACE_BIOLATENCY_RESOURCE_NAME
+  );
+}
+
+function getLatencyResult(cb: (data: any) => void) {
+  return ResourceAPI.apiEndpoint.get(
+    IG_TRACE_NAMESPACE,
+    TRACE_BIOLATENCY_RESOURCE_NAME,
+    function (data: any) {
+      cb(data);
+    }
+  );
+}
+
+const LATENCY_CHECK_NOT_STARTED = -1;
+const LATENCY_CHECK_STOPPED = 0;
+const LATENCY_CHECK_STARTED = 1;
+
+function CustomComponent(props: any) {
+  const { resource } = props;
+  const [outputString, setOutputString] = React.useState('');
+  const [outputError, setOutputError] = React.useState(null);
+  const [latencyCheckStatus, setLatencyCheckStatus] = React.useState(LATENCY_CHECK_NOT_STARTED);
+  let componentToRender = (
+    <LatencyCheckInitial
+      setLatencyCheckStatus={() => setLatencyCheckStatus(LATENCY_CHECK_STARTED)}
+    />
+  );
+  if (resource && resource.kind === 'Node') {
+    switch (latencyCheckStatus) {
+      case 0:
```

Comment:
Use enum values (after making it an enum) for the comparison. It will make the code much more readable.